### PR TITLE
Updating ReadMe file

### LINKED ISSUE #4: Updating ReadMe file
Updating the ReadMe file to describe more thoroughly the 'picATrip' project.

### PR DESCRIPTION
**Pull request by maayanhd**
- Updated [ReadMe.md][1] file to describe more thoroughly the 'picATrip' project.

Closes #4
Added general system functionality description to file

### CODE REVIEW

**Review by nadavspinco — APPROVED** [review]

**Review by nadavspinco — APPROVED** [review]

**Comment by EdDev** [comment]
> related issue: #4

You can automatically link the issue to the PR, see [here](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

File: README.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
 # picATrip
 picATrip - Beyond 05, team 2 project
+
+A platform for sharing experiences from trips in a form of photos based on various parameters as locations/duration time/difficulty, and provides a trip search functionality and commenting functionality. 
+It allows users to have the oppurtunity to contribute their trip experience and thoughts as well as learn about trips of others across the country, and adding comment if desired.
```
Comment (Yarboa) [inline README.md:5]: I would advise to use some markdown formats to the content
https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet
Such as:
What is picAtrip?
Who are we?

**Comment by Leeadar** [comment]
The content looks great!
You can also add that the project runs on Vagrant VM (Fedora)
and uses Django as a web framework.

**Comment by maayanhd** [comment]
@Leeadar, I'm currently working on a fix, just opened a new issue #11 to address this part also.

**Review by nadavspinco — APPROVED** [review]

**Pull request merged**